modules/claims_api/app/controllers/claims_api/v1/forms/power_of_attorney_controller.rb

```diff
@@ -15,7 +15,10 @@ class PowerOfAttorneyController < ClaimsApi::V1::Forms::Base
           permit_scopes %w[claim.read] if request.get?
         end
         before_action { permit_scopes %w[claim.write] if request.post? || request.put? }
-
+        before_action except: %i[status active] do
+          request&.headers&.to_h.present?
+          @req_headers = request&.headers&.to_h
+        end
```

Comment:
could we not use `header_request?`?

Reply:
also, I think the goal is to do something like the `permit_scopes`..... pseudo below

```
before_action { check_request_ssn_matches_mpi except: %w[status active] if header_request? }
```

This currently throws the SSN error on every request without headers.